catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/SortedServiceList.java

```diff
@@ -93,7 +93,9 @@ public void bindPlugin(ServiceReference ref) {
       LOGGER.debug("BundleContext was null, unable to add service reference");
     }
 
-    LOGGER.debug(Arrays.asList(serviceMap.values()).toString());
+    if (LOGGER.isDebugEnabled()) {
+      LOGGER.debug(Arrays.asList(serviceMap.values()).toString());
```

Comment:
✏️ Is `.toString()` required here and L115?

Reply:
Single argument logging methods require a string or a throwable.